<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleDialog</class>
 <widget class="QDialog" name="VehicleDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление транспортного средства</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 199);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="Group5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0" colspan="3">
     <widget class="QPushButton" name="btnCancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Год выпуска</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="txtPrice"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="txtYear"/>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPushButton" name="btnOk">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Описание</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Модель</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Марка</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="txtModel"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="txtDescription"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="txtStatus"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Статус</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="cmbMark"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
